example/lib/screens/preference_center.dart

```diff
@@ -130,10 +129,10 @@ class _PreferenceCenterState extends State<PreferenceCenter>
       title: Text('${item.display.title}',
           style: TextStyle(fontWeight: FontWeight.bold)),
       subtitle: Text('${item.display.subtitle}'),
-      value: isSubscribedContactSubscription(item.subscriptionId, []),
+      value: isSubscribedContactSubscription(item.subscriptionId, ["app"]),
```

Comment:
I think the scopes array here should be read from the `item`, but it looks like we may have missed the `scopes` field?

Here's where that's defined in the contact subscription items on [Android](https://github.com/urbanairship/android-library-dev/blob/main/urbanairship-preference-center/src/main/java/com/urbanairship/preferencecenter/data/Item.kt#L161) and [iOS](https://github.com/urbanairship/ios-library-dev/blob/main/Airship/AirshipPreferenceCenter/Source/data/Items.swift#L221).

It looks like `isSubscribedContactSubscription` will do the right thing if we pass it the item scopes.